pkg/ui/src/util/eventTypes.ts

```diff
@@ -44,7 +46,8 @@ export const SET_CLUSTER_SETTING = "set_cluster_setting";
 export const nodeEvents = [NODE_JOIN, NODE_RESTART, NODE_DECOMMISSIONED, NODE_RECOMMISSIONED];
 export const databaseEvents = [CREATE_DATABASE, DROP_DATABASE];
 export const tableEvents = [CREATE_TABLE, DROP_TABLE, ALTER_TABLE, CREATE_INDEX,
-  DROP_INDEX, CREATE_VIEW, DROP_VIEW, REVERSE_SCHEMA_CHANGE, FINISH_SCHEMA_CHANGE];
+  DROP_INDEX, CREATE_VIEW, DROP_VIEW, REVERSE_SCHEMA_CHANGE, FINISH_SCHEMA_CHANGE,
+  FINISH_SCHEMA_CHANGE_ROLLBACK];
```

Comment:
ugh, I don't like this formatting but w/e don't worry about it

Reply:
Mind that I haven't linted (and this may undoubtedly not even compile). -manyellsatcloud-

Reply:
no I think we just have permissive linting for this case